pkg/network/ovn_kubernetes.go

```diff
@@ -50,13 +51,15 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["CNIBinDir"] = CNIBinDir
 	data.Data["OVN_NB_PORT"] = OVN_NB_PORT
 	data.Data["OVN_SB_PORT"] = OVN_SB_PORT
+	data.Data["OVN_DB_SVC"] = OVN_DB_SVC
 	data.Data["OVN_NB_RAFT_PORT"] = OVN_NB_RAFT_PORT
 	data.Data["OVN_SB_RAFT_PORT"] = OVN_SB_RAFT_PORT
-	data.Data["OVN_NB_DB_LIST"] = dbList(bootstrapResult.OVN.MasterIPs, OVN_NB_PORT)
-	data.Data["OVN_SB_DB_LIST"] = dbList(bootstrapResult.OVN.MasterIPs, OVN_SB_PORT)
-	data.Data["OVN_NB_ADDR_LIST"] = addrList(bootstrapResult.OVN.MasterIPs, OVN_NB_PORT)
-	data.Data["OVN_SB_ADDR_LIST"] = addrList(bootstrapResult.OVN.MasterIPs, OVN_SB_PORT)
+	data.Data["OVN_NB_DB_LIST"] = "ssl:" + net.JoinHostPort(OVN_DB_SVC, OVN_NB_PORT)
+	data.Data["OVN_SB_DB_LIST"] = "ssl:" + net.JoinHostPort(OVN_DB_SVC, OVN_SB_PORT)
+	data.Data["OVN_NB_ADDR_LIST"] = "ssl://" + net.JoinHostPort(OVN_DB_SVC, OVN_NB_PORT)
+	data.Data["OVN_SB_ADDR_LIST"] = "ssl://" + net.JoinHostPort(OVN_DB_SVC, OVN_SB_PORT)
```

Comment:
Not sure it really makes sense to template this stuff in if it is effectively hardcoded now

Reply:
I prefer hardcoding it in go rather than in bash, given the ipv4/ipv6 difference for host-port.

Reply:
there's no ipv4/ipv6 difference any more though because you're using a hostname instead of an IP

Reply:
OK, I got confused with the brackets around ipv6 addresses and thought it applied to ports too somehow. Honestly though I prefer keeping it templated here in case we ever change the service name. Would be less places in the code to change.